lib/pretty/style.ml

```diff
@@ -6,3 +6,6 @@ let directory = ANSITerminal.[ Bold; magenta ]
 let chosen = ANSITerminal.[ Bold; magenta ]
 let secondary = ANSITerminal.[ cyan ]
 let bold = ANSITerminal.[ Bold ]
+let pr_closed = ANSITerminal.[ Bold; red ]
+let pr_open = ANSITerminal.[ Bold; green ]
+let pr_merged = ANSITerminal.[ Bold; magenta ]
```

Comment:
Great stuff!

Semantic names are really great for such styles, as they make things easier.

Side note, this `Style` module does the job for now but I plan to revamp it in the future. Main reasons:

1. Not enough colours
2. Not composable enough (I have to create a style each time instead of easily composing)

I haven't finalised the design, something to think about. But I'll open an issue with my thoughts once I have more clarity 🙂